docs/getting-started/install-linux.md

```diff
@@ -32,6 +32,8 @@ $ curl -sL https://github.com/inspektor-gadget/inspektor-gadget/releases/downloa
 $ ig version
 ```
 
+We strongly advised you to verify `ig` by following our [detailed instructions](./verify.md#verify-an-asset).
```

Comment:
```suggestion
We strongly advised you to verify `ig` by following our [detailed instructions](./verify.md#Verify-an-asset).
```

otherwise it doesn't forward me to the section.